src/utils/omitUndefined.ts

```diff
@@ -0,0 +1,11 @@
+import _ from 'lodash';
```

Comment:
Does it matter whether one imports the default export like this, or imports the specific functions individually?

```suggestion
import { isUndefined, omitBy } from 'lodash';
```

Does it affect the bundle size?

Reply:
As I'm not so experienced in the node ecosystem, I tried consulting some LLM about this:

> `import _ from "lodash";`
This imports the entire Lodash library. Even if you only use one or two functions, your bundle will include all of Lodash. This results in a much larger bundle size.

> `import { isUndefined, omitBy } from "lodash";`
This is slightly better because you’re only importing the specific functions you need. However, with Lodash, this doesn’t guarantee tree-shaking (automatic removal of unused code) in some bundlers. Depending on the build setup (e.g., Webpack, Rollup), you might still end up with unnecessary parts of Lodash included in the bundle.